pkg/lib/web/static_assets.go

```diff
@@ -22,6 +22,8 @@ var AuthgearDarkThemeCSS = resource.RegisterResource(CSSDescriptor{
 var AppLogo = resource.RegisterResource(ImageDescriptor{Name: "app_logo"})
 var AppLogoDark = resource.RegisterResource(ImageDescriptor{Name: "app_logo_dark"})
 var Favicon = resource.RegisterResource(ImageDescriptor{Name: "favicon"})
+var AppBackgroundImage = resource.RegisterResource(ImageDescriptor{Name: "app_background_image"})
+var AppBackgroundImageDark = resource.RegisterResource(ImageDescriptor{Name: "app_background_image_dark"})
```

Comment:
Not sure how to increase file size limit for background image

Reply:
Seems no easy way at the moment :sosad:
But it make sense that background image could be larger than 100KB? @fungc-io

Reply:
I suggest to add an interface:
```golang
type SizeLimitDescriptor interface {
  GetSizeLimit() int
}
```

And move the logic that checks the size limit (Search for `ConfigFileMaxSize`) into `Manager.applyUpdates(...)`.
After `manager.Resolve(u.Path)` called, you got the descriptor. Check if it implements `SizeLimitDescriptor`. If yes, use `GetSizeLimit` to get the limit and check the size. Else, check with `ConfigFileMaxSize` as the default limit.

Reply:
I think I will create another ticket for it. This PR is already too big